portfolio/src/main/webapp/index.html

```diff
@@ -129,8 +127,11 @@ <h2 id="outro">...and here we are today.</h2>
                         <h2>Previous Comments</h2>
                         <hr/>
                         <div id="msg-container"></div>
+                        <script>getComments(5)</script>
```

Comment:
Will this always return 5 comments?

Reply:
I set it so that initially, by default, 5 comments will be returned. But that's only initial loading; once you manually restrict number of comments, it will change to reflect your selection.

Reply:
I see. I got the same problem with Razvan that changing the number doesn't affect how many comments are shown up. Seems the onchange below is not working as expected.

Reply:
Fixed it in the new commit!